frontend/src/app/about/page.module.css

```diff
@@ -3,22 +3,34 @@
     margin: 20px;
 }
 
+.container {
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+.container p, .link {
+    padding: 5px;
+}
+
+.section {
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
 .creator_name {
-    color: #94a3b8;
+    color: #626a73;
 }
 
 .subheading {
-    color: #f472b6;
+    color: hotpink;
     margin: 10px;
     text-align: center;
 }
 
 .profile_photo {
```

Comment:
Could: The profile photos have rounded corners or be circles?
Could you also try the photos as a smaller size and see if it makes the page looks better, as there's not much content next to them? :)

Reply:
Have made smaller and circular